### PR TITLE
clone vs init

### PR DESCRIPTION
**Pull request by naderkhalil**
Changed `init`-ing an existing project to `clone`. Fixed some output too.

### CODE REVIEW

**Comment by naderkhalil** [comment]
Just added a commit for help text when logging in:

image

File: internal/initialize/command.go [hunk 1]
```diff
@@ -73,14 +67,44 @@ func NewCmdInit(t *terminal.Terminal) *cobra.Command {
...
+
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Initialize a Brev Project",
+		Long: `Use this to initialize a Brev project. Ex:
+		
+		// Init a new project in current directory
+		brev init
+	
+		`,
```
Comment (drewmalin) [inline internal/initialize/command.go:89]: Do you mind converting this to use the 'Example' built-in? I swept through in the other PR to make consistent use of the usage text
```suggestion
		Long: "Use this to initialize a Brev project."
		Example: `  // Init a new project in current directory
  brev init`,
```

File: internal/initialize/command.go [hunk 2]
```diff
@@ -15,28 +15,21 @@ import (
...
-		Short:       "Initialize a Brev Project",
-		Long:        "Use this to initialize a Brev project.",
-		Example: `  // To init new project in current directory
-  brev init
-
-  // To init existing project
-  brev init <project_name>`,
+		Long:        "Clone an existing Brev project",
+		Example: ` // To clone your existing Brev project
+		brev clone --name your_project_name`,
```
Comment (drewmalin) [inline internal/initialize/command.go:27]:
```suggestion
		Example: `  // To clone your existing Brev project
  brev clone --name your_project_name`,
```

This is annoying, but within the tick marks go with preserve the string exactly as you provide. So the helptext here looks like:

```
Initialize a Brev project.

Usage:
  brev init [flags]

Examples:
 // To init new project in current directory
  			brev init
```

With the above change you'll get:

```
Clone an existing Brev project

Usage:
  brev clone [flags]

Examples:
  // To clone your existing Brev project
  brev clone --name your_project_name
```

File: internal/initialize/command.go [hunk 3]
```diff
@@ -72,14 +65,42 @@ func NewCmdInit(t *terminal.Terminal) *cobra.Command {
...
 
+func NewCmdInit(t *terminal.Terminal) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:         "init",
+		Annotations: map[string]string{"project": ""},
+		Short:       "Initialize a Brev Project",
+		Long:        "Initialize a Brev project.",
+		Example: `  // To init new project in current directory
+  			brev init`,
```
Comment (drewmalin) [inline internal/initialize/command.go:85]: Similar problem as above

**Review by drewmalin — APPROVED** [review]

**Pull request merged**